api/src/routes/events.ts

```diff
@@ -23,9 +23,21 @@ eventRouter.post("/", [auth], async (req: Request, res: Response) => {
     lng,
     repeat,
     notes,
+    startFormattedAddress,
+    startLat,
+    startLng,
   } = req.body;
 
-  if (!name || !formattedAddress || !start_time || !end_time || !repeat) {
+  if (
```

Comment:
why are you checking if startLat and startLng are filled? are those filled automatically?